torch/_dynamo/variables/builder.py

```diff
@@ -1103,7 +1103,6 @@ def wrap_tensor(self, value: torch.Tensor):
             return TensorWithTFOverrideVariable.create(
                 self.tx,
                 tensor_variable,
-                source,
```

Comment:
Why remove this?

Reply:
the source will be the same as the tensor_variable (it wasn't used anywhere). In the next PR, TensorWithTFOverrideVariable will inherit from TensorVariable so this will make sense.